src/services/ansiblePlaybook.ts

```diff
@@ -0,0 +1,179 @@
+import * as child_process from 'child_process';
+import { ExecException } from 'child_process';
+import { URL } from 'url';
+import { promisify } from 'util';
+import {
+  Connection,
+  Diagnostic,
+  DiagnosticSeverity,
+  Position,
+  Range,
+} from 'vscode-languageserver';
+import { TextDocument } from 'vscode-languageserver-textdocument';
+import { withInterpreter } from '../utils/misc';
+import { WorkspaceFolderContext } from './workspaceManager';
+const exec = promisify(child_process.exec);
```

Comment:
I'd have this separated from imports too.

Reply:
Me too. I will update it. Thanks.

Reply:
Updated! Thanks for the suggestion.